packages/plugin-page-view-tracking/src/plugin-campaign-tracker.ts

```diff
@@ -0,0 +1,53 @@
+import { AttributionBrowserOptions, Campaign } from '@amplitude/analytics-types';
+import { CampaignTracker } from '@amplitude/analytics-client-common';
+import { BASE_CAMPAIGN } from './constant';
+import { Storage } from '@amplitude/analytics-types';
+
+export class PluginCampaignTracker {
```

Comment:
also feels weird that we have campaign tracker in page view tracking since this plugin does not track campaigns. might just be a naming concern